Dockerfile

```diff
@@ -1,19 +1,22 @@
 ####################################################################
-# base image contains all the dependencies we need to build the code
-FROM valhalla/valhalla:build-3.1.5 as builder
+FROM ubuntu:20.04 as builder 
 MAINTAINER Kevin Kreiser <kevinkreiser@gmail.com>
 
-# get the code into the right place and prepare to build it
+# set paths
+ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH
+ENV LD_LIBRARY_PATH /usr/local/lib:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/lib32:/usr/lib32
+
+# install deps
 WORKDIR /usr/local/src/valhalla
+COPY ./scripts/install-linux-deps.sh /usr/local/src/valhalla/scripts
+RUN bash ./scripts/install-linux-deps.sh
```

Comment:
first copy and execute the deps script for better layer caching as @ImreSamu pointed out. this is effectively doing a similar job as having a separate build docker image, esp if (hopefully soon) remote layer caching is standard by default